### PR TITLE
Bump BOSH, UAA, and postgres

### PR DESCRIPTION
**Pull request by tjliebel**

### CODE REVIEW

File: ci/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Software Updates
+
+- Updated BOSH to 270.11.0 from 270.10.0
+- Updated Azure CPI to 37.1.0 from 37.0.0
```
Comment (thomasmitchell) [inline ci/release_notes.md]: add uaa in here.

File: ci/release_notes.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- Updated Postgres job to 10 from 9.4
```
Comment (thomasmitchell) [inline ci/release_notes.md]: move the postgres job change to a section for Component Changes

File: ci/release_notes.md (hunk 1 above)
Comment (tjliebel) [reply ci/release_notes.md]: Added.

File: ci/release_notes.md (hunk 2 above)
Comment (tjliebel) [reply ci/release_notes.md]: Added that section and moved the update there.

**Pull request merged**